mantidproject/autoreduce
# Ability to re-run multiple jobs at a time

Fixes #81 

To test:
* Go to the submit runs page of an instrument
* Type in a start and end run that includes runs that have already been run
* Confirm that these runs are re-run

Also to check:
* Descriptions are given to each run
* No more than 20 runs can be re-run at once
* Variables on runs are as expected
